services/actions/notifier.go

```diff
@@ -623,6 +629,10 @@ func (n *actionsNotifier) UpdateRelease(ctx context.Context, doer *user_model.Us
 }
 
 func (n *actionsNotifier) DeleteRelease(ctx context.Context, doer *user_model.User, rel *repo_model.Release) {
+	if rel.IsTag {
```

Comment:
Is this really right? @wolfogre

Reply:
I think this function make sense:
https://github.com/go-gitea/gitea/blob/17d7ab5ad4ce3d0fbc1251572c22687c237a30b1/services/release/release.go#L305-L370

``` golang
if delTag {
  ......
  notify_service.PushCommits()
  notify_service.DeleteRef()
}
......
if !rel.IsDraft {
  notify_service.DeleteRelease(ctx, doer, rel)
}
```

Reply:
I think it's fine, it doesn't affect the actual usage scenario.